modules/countdown-timer/includes/EnqueueScript.php

```
@@ -73,22 +77,22 @@ public function admin_enqueue_scripts( $hook ) {
 			return;
 		}
 
-		$settings_file = require sgsb_modules_path( 'CountdownTimer/assets/build/settings.asset.php' );
-		$style_file    = require sgsb_modules_path( 'CountdownTimer/assets/build/settings.asset.php' );
+		$settings_file = require PluginHelper::get_modules_path( 'countdown-timer/assets/build/settings.asset.php' );
+		$style_file    = require PluginHelper::get_modules_path( 'countdown-timer/assets/build/settings.asset.php' );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$style_file'. (Unused Code Rules)

(UnusedLocalVariable)